packages/zoe/test/unitTests/contracts/zcfTesterContract.js

```diff
@@ -7,7 +7,14 @@ import '../../../exported';
  * @type {ContractStartFn}
  */
 const start = zcf => {
-  return { creatorFacet: zcf };
+  const publicFacet = harden({
```

Comment:
Hmm, has this been rebased recently? I think this file is in a completely different location now.